Services/Database/classes/class.ilDBConstants.php

```diff
@@ -85,7 +83,6 @@ class ilDBConstants
         ilDBConstants::TYPE_POSTGRES => "Postgres (experimental)",
         ilDBConstants::TYPE_GALERA => "Galera-Cluster (experimental)",
         // Development identifiers (will be removed in 5.3)
-        ilDBConstants::TYPE_PDO_MYSQL_MYISAM => "MySQL 5.5.x or higher (MyISAM engine) [developers-identifier]",
         ilDBConstants::TYPE_PDO_MYSQL_INNODB => "MySQL 5.5.x or higher (InnoDB engine) [developers-identifier]",
```

Comment:
we can change this like this:
```suggestion
        ilDBConstants::TYPE_PDO_MYSQL_INNODB => "MySQL 5.7.x or higher (InnoDB engine)",
```

Reply:
Done.